src/renderer/views/wallet/SendView.tsx

```diff
@@ -24,12 +26,25 @@ const SendView: React.FC<Props> = (): JSX.Element => {
 
   const intl = useIntl()
 
-  const { transaction: transactionService, balancesState$, explorerUrl$ } = useBinanceContext()
+  const { transaction: transactionService, balancesState$, explorerUrl$, client$ } = useBinanceContext()
   const balancesState = useObservableState(balancesState$, initial)
   const explorerUrl = useObservableState(explorerUrl$, O.none)
+  const client = useObservableState<O.Option<BinanceClient>>(client$, O.none)
+
   const balances = useMemo(() => toAssetWithBalances(balancesState, intl), [balancesState, intl])
   const oSelectedAssetWB = useMemo(() => getAssetWithBalance(balances, oSelectedAsset), [oSelectedAsset, balances])
 
+  const addressValidation = useMemo(
+    () =>
+      FP.pipe(
+        client,
+        O.map((c) => c),
```

Comment:
seems like useless :)

Reply:
good catch - just an artifact of debugging ;)